models/sipnet/R/read_restart.SIPNET.R

```diff
@@ -102,6 +102,13 @@ read_restart.SIPNET <- function(outdir, runid, stop.time, settings, var.names, p
     params$restart["litter_mass_content_of_water"] <- ens$litter_mass_content_of_water[last]
   }
   
+  if ("SoilMoist" %in% var.names) {
+    forecast[[length(forecast) + 1]] <- ens$SoilMoist[last]
```

Comment:
Is read.output already providing the SoilMoist you need? Based on prior discussion I thought it would need to be manually calculated from SoilMoistFrac